blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -65,6 +65,20 @@ def sql(url):
             drop(t)
 
 
+@pytest.yield_fixture
```

Comment:
I wanted more data than existing fixtures have (so that I can filter by it and test relabeling over a selection).